__tests__/erebos-api-bzz-browser.js

```diff
@@ -2,15 +2,29 @@
  * @jest-environment jsdom
  */
 
+import fetch from 'node-fetch'
```

Comment:
Don't the tests work without importing it?
Ideally it should be provided by JSDOM, otherwise better use a mock I think, otherwise the tests might have false-positive using APIs provided by `node-fetch` but that wouldn't be provided by the browser.